app/models/user.rb

```diff
@@ -86,6 +86,14 @@ def active_profile
     profiles.find(&:active?)
   end
 
+  # This user's most recently activated profile that has also been deactivated
```

Comment:
Well, on second thought, this is a data accessing method, just like `active_profile` so I think it makes sense here? How strongly do you feel?